roll.go

```diff
@@ -13,7 +13,7 @@ import (
 
 const UintBytes = 2
 
-var diceRegexp = regexp.MustCompile(`(?i)^\/*(\d+)d(\d+)$`)
+var diceRegexp = regexp.MustCompile(`(?i)^(\d+)d(\d+)$`)

```

Comment:
This change is likely to break the slack_roll functionality.

I made this change so that this regex can be used outside the context of a URL path.